newIDE/app/src/MainFrame/Preferences/PreferencesProvider.js

```diff
@@ -58,7 +60,7 @@ export default class PreferencesProvider extends React.Component<Props, State> {
           language,
         },
       }),
-      () => this._persistValuesToLocalStorage(this.state)
+      () => this._persistValuesToLocalStorage(this.state.values)
```

Comment:
I would recommend to cancel these changes to reduce the PR size if they are not needed